### PR TITLE
feat: discord bot

### PR DESCRIPTION
**Pull request by cachho**
## Description

Haven't had a chance to test this since the commands won't sync.

Documentation updates are also missing

Fixes #

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Please delete options that are not relevant.

- [ ] Unit Test
- [ ] Test Script (please provide)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] Made sure Checks passed

### CODE REVIEW

**Comment by cachho** [comment]
one thing that I did with my discord bot is include the original message, because the slash command hides it. I will make an arg param for that.

**Comment by cachho** [comment]
Through testing I found out that the only permission needed is send message.  Applications command scope also isn't required

File: docs/examples/discord_bot.mdx [hunk 1]
```diff
@@ -8,40 +8,51 @@ title: '🤖 Discord Bot'
...
-- 🎉 The bot has been successfully added to your server.
+- 🎉 The bot has been successfully added to your server. But it's still offline.
 
-### 🐳 Docker Setup
+### Take the bot online
+
+1. Install embedchain python package:
+
+```bash
+pip install embedchain
```
Comment (taranjeet) [inline docs/examples/discord_bot.mdx:29]: we dont need as we are already installing "embedchain[discord]"

**Review by taranjeet — APPROVED** [review]

File: docs/examples/discord_bot.mdx (hunk 1 above)
Comment (cachho) [reply docs/examples/discord_bot.mdx:29]: done

**Pull request merged**